manoseimas/lobbyists/models.py

```diff
@@ -29,6 +29,26 @@ class Lobbyist(CrawledItem):
     def __unicode__(self):
         return self.name
 
+    def get_clients(self):
+        clients = []
+        for declaration in self.lobbyistdeclaration_set.all():
+            clients.extend(declaration.clients.all())
+        return clients
```

Comment:
I've a feeling this ought to be doable with a single query instead of many.

If there are multiple declarations made on behalf of the same client, this will return duplicate list items.  Is that good?